packages/core/src/components/WarningPanel/WarningPanel.stories.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import WarningPanel from '.';
+import { Link, Button } from '@material-ui/core';
+
+export default {
+  title: 'Warning Panel',
+  component: WarningPanel,
+};
+
+export const Default = () => (
+  <WarningPanel
+    title="Lineage Warning"
+    message={
+      <>
+        This entity is missing lineage data. If this is unexpected, please make
+        sure you have set up Lineage correctly by following{' '}
```

Comment:
Question (feel free to ignore) : What is missing "lineage data"?

Is this just an example of any kind of warning? Perhaps the warning text can be simplified.